scalecube-organization/src/test/java/io/scalecube/organization/Base.java

```diff
@@ -102,21 +94,17 @@
   protected String organizationId;
   protected Organization organisation;
   protected Token token = new Token("user1");
-  protected Repository<Organization, String> organizationRepository;
-  protected UserOrganizationMembershipRepository orgMembersRepository;
-  private OrganizationMembersRepositoryAdmin admin;
+  protected OrganizationsRepository organizationRepository;
 
   protected Base() {
-    orgMembersRepository = new InMemoryUserOrganizationMembershipRepository();
     organizationRepository = new InMemoryOrganizationRepository();
-    admin = new InMemoryOrganizationMembersRepositoryAdmin();
     service = createService(testProfile);
     new File("keystore.properties").deleteOnExit();
   }
 
   @BeforeAll
   static void beforeAll() throws NoSuchAlgorithmException {
-    keyPairGenerator = KeyPairGenerator.getInstance("RSA");
+    KeyPairGenerator keyPairGenerator = KeyPairGenerator.getInstance("RSA");
```

Comment:
why did you remove this static member?

Reply:
Removed it at all. It is not used in these tests.